tsconfig.json

```diff
@@ -14,37 +14,35 @@
     "noUnusedParameters": false,
     "noImplicitReturns": true,
     "noEmitOnError": false,
-    "noEmit": true,
+    "declaration": true,
     "inlineSourceMap": true,
     "inlineSources": true,
-    "baseUrl": ".",
-    "module": "es6",
+    "module": "esnext",
```

Comment:
why is this `esnext`? I would have thought we'd use either `ES2020` or `ES2015`... 🤔 

https://www.typescriptlang.org/tsconfig#module